packages/ses/src/tame-locale-methods.js

```diff
@@ -0,0 +1,54 @@
+import { assert } from './assert.js';
+import { getOwnPropertyNames, defineProperty } from './commons.js';
+
+const localePattern = /^(\w*[a-z])[L]ocale([A-Z]\w*)$/;
```

Comment:
Is [L] intended to be alone in that group?

Reply:
Good catch!